swagger.yml

```
@@ -0,0 +1,137 @@
+swagger: '2.0'
+
+# This is your document metadata
+info:
+  version: "0.0.0"
+  title: Communication_Notebook
+
+# Describe your paths here
+paths:
+  # This is a path endpoint. Change it.
+  /kid/{kidId}:
+    # This is a HTTP operation
+    get:
+      tags:
+      - "kid"
+      # Describe this verb here. Note: you can use markdown
+      summary: "子供の名前、年齢取得"
+      description: ""
+      # This is array of GET operation parameters:
+      parameters:
+        # An example parameter that is in query and is required
+        -
+          name: kidId
+          in: path
+          description: 特定の子供の名前と年齢を取得
+          required: true
+          type: integer
+          format: int64
+      # Expected responses for this operation:
+      responses:
+        # Response code
+        200:
+          description: 成功時のレスポンス
+          # A schema describing your response object.
+          # Use JSON Schema format
+          schema:
+            title: 子供の年齢と名前を返す
+            type: object
+            properties:
+              name:
+                type: string
+              age:
+                type: integer
+  /communication_notebook/{kidId}?{date=Date}:
+    get:
+      tags:
+      - "communication_notebook"
+      # Describe this verb here. Note: you can use markdown
+      summary: "ある日付の連絡帳の詳細表示"
+      description: ""
+      # This is array of GET operation parameters:
+      parameters:
+        # An example parameter that is in query and is required
+        - name: kidId
```

```suggestion
        - name: kid_id
```

アッパーケースとスネークケースを混合しないでください